variables.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+#Promenade Variables
+DOCKER_VERSION=1.12.6-0ubuntu1~16.04.1
```

Comment:
In line with the above comments about having only one config file, I think we could actually put this at the top of the script for now and use a default for users to override:

```bash
DOCKER_VERSION=${DOCKER_VERSION:-1.12.6-0ubuntu1~16.04.1}
```